README.md

```diff
@@ -40,6 +40,11 @@ please:
 When contributing SystemVerilog source code, please try to be consistent and adhere to [our Verilog
 coding style guide](https://github.com/lowRISC/style-guides/blob/master/VerilogCodingStyle.md).
 
+When contributing C or C++ source code, please try to adhere to [the OpenTitan C++ coding style
+guide](https://docs.opentitan.org/doc/rm/c_cpp_coding_style/).
+All C and C++ code should be formatted with clang-format before committing.
+Either run clang-format -i `filename.cc` or git clang-format-6.0 on added files.
```

Comment:
Please format the whole command line monospace.
`git clang-format-6.0` -> `git clang-format`